CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+# Contributing to DAML
```

Comment:
This looks like you copied the `CONTRIBUTING.md` page from the `daml` repo. This needs to be adapted to `ghc-lib`.